policyengine_us/parameters/gov/local/ny/nyc/tax/income/credits/household/flat_amount.yaml

```
@@ -0,0 +1,22 @@
+description: NYC provides a household credit of this fixed amount to single filers, based on recomputed federal AGI.
+metadata:
+  label: NYC Household Credit Fixed Amount for single filers
+  type: single_amount
+  threshold_unit: currency-USD
+  amount_unit: currency-USD
+  reference:
+    - title: Instructions for Form IT-201
+      href: https://www.tax.ny.gov/pdf/2022/printable-pdfs/inc/it201i-2022.pdf#page=17
+brackets:
+  - threshold:
+      2022-01-01: 0
+    amount:
+      2022-01-01: 15
+  - threshold:
+      2022-01-01: 10_001
```

And other thresholds, since that's the number in the law. Instead use `right=True` in the `calc` functions per https://openfisca.org/doc/coding-the-legislation/legislation_parameters.html#creating-scales
```suggestion
      2022-01-01: 10_000
```